From c47dd9c83b4e48f87d3f1a0b8eaadb5b6d3d3aa0 Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Sat, 13 Apr 2019 06:23:01 +0000
Subject: [PATCH 114/155] HACK: silence "PFNs busy" log spam

Even with ratelimiting in https://lore.kernel.org/patchwork/patch/816616/ lots
of "PFNs busy"  messages spam the system log, so silence the message.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 mm/page_alloc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mm/page_alloc.c b/mm/page_alloc.c
index 15c2050c629b..f66735ed0a4c 100644
--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -8467,8 +8467,6 @@ int alloc_contig_range(unsigned long start, unsigned long end,
 
 	/* Make sure the range is really isolated. */
 	if (test_pages_isolated(outer_start, end, false)) {
-		pr_info_ratelimited("%s: [%lx, %lx) PFNs busy\n",
-			__func__, outer_start, end);
 		ret = -EBUSY;
 		goto done;
 	}
-- 
2.17.1

